.github/workflows/rfs_pr_e2e_test.yml

```
@@ -0,0 +1,40 @@
+name: Jenkins E2E
```

```suggestion
name: Jenkins
```
E2E isn't really required here and uses acronyms that folks might not be familiar with.